tools/dissector/packet-opendds.cpp

```diff
@@ -828,41 +940,86 @@ namespace OpenDDS
                 }
         },
         { &hf_sample_content_filt,
+            { "Content Filters",
```

Comment:
Is this the boolean flag for content filter?  Maybe change the name the "Content Filters" to something else since the one below also uses that.